src/extension.ts

```diff
@@ -99,7 +99,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         createFunction: createFunctionFromApi,
         downloadAppSettings: downloadAppSettingsFromApi,
         uploadAppSettings: uploadAppSettingsFromApi,
-        apiVersion: '1.7.0'
+        apiVersion: '1.8.0'
```

Comment:
I think it does _not_ need to be changed, but does this need to be bumped to 1.9.0 in `main`?

Reply:
I think the API would remain the same because there'd be no changes to the API between 1.6.3 and whatever we release in main.

Reply:
I thought so but wasn't 100%.